features/polyExplorer/src/screens/stories/story-utils.js

```diff
@@ -1,4 +1,4 @@
-import i18n from "../../i18n.js";
+import i18n from "!silly-i18n";
```

Comment:
I really keep not seeing the tradeoffs here. You need to do a very extensive refactoring, substuting every single piece of import from here; this is mainly to use that single string as an indicator this line needs to be substituted. But all lines you have changed follow the same pattern: essentially a path that ends with `i18n.js`; that's a regex that can easily be detected, and you wouldn't need to refactor sources anywhere.
Also, you delete the original `i18n.js` file without any need to do so, it's a perfectly good file, not a lot going for it in this case, but good enough anyway. If you leave it there, it's not going to be used in the bundle; fair enough, but you don't need to ditch it in that case. It can be used as a mock for unbundled tests, anyway.